tests/ai_safety/evalhub/kueue/test_evalhub_kueue_negative.py

```
@@ -158,3 +175,151 @@ def test_forbidden_cross_tenant_access(
+                            namespace=evalhub_kueue_namespace.name,
+                            evalhub_job_id=job2_id,
+                        )
+                    except TimeoutExpiredError:
+                        log_job_kueue_labels(admin_client, evalhub_kueue_namespace.name, job2_id)
+                        raise
+
+                    completed_ids: set[str] = set()
+                    offset = 0
+                    page_size = 50
+                    max_pages = 20
+                    for page in range(max_pages):
```

Ruff (0.15.21)

[warning] Loop control variable `page` not used within loop body

Rename unused `page` to `_page`

(B007)